front/app/containers/Admin/dashboard/users/Charts/BarChartByCategory.tsx

```diff
@@ -58,6 +58,7 @@ interface InputProps {
   className?: string;
   customId?: string;
   xlsxEndpoint: string;
+  interactive?: boolean;
```

Comment:
I would rename this to `showExportMenu` or something like that- `interactive` requires you to understand that this refers to the fact this component can be used in an 'interactive' (inside of the dashboard) or 'non-interactive' (inside of the report) context. That is not something you should have to worry about if you're looking at the component code

Reply:
ha ha, I already had it as that at some point and then changed it! Fair point though